samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/transition.c

```diff
@@ -507,6 +602,11 @@ K_WORK_DEFINE(light_lightness_linear_work, light_lightness_linear_work_handler);
 K_WORK_DEFINE(light_ctl_work, light_ctl_work_handler);
 K_WORK_DEFINE(light_ctl_temp_work, light_ctl_temp_work_handler);
 
+void rt_handler(struct k_timer *dummy)
+{
+	rt_counter--;
+}
```

Comment:
Wouldn't it be more power efficient to just store the "start" as a time-stamp and then do the subtraction based on the elapsed time when it's actually needed, instead of having this timer firing off every 100ms?

Reply:
Timer of 100ms will turn off with Transition End.

Reply:
Timer overflow is going to be issue ....if we consider the way you have suggested

Reply:
Please wait ...don't review ...since I'm correcting the things.

Reply:
I'm trying to implement as per your suggetion

Reply:
Done !!
I've implemented as per your suggestion. Yes it is accurate or correct implementation than mine !!